pyproject.toml

```diff
@@ -93,10 +94,11 @@ epacems_to_parquet = "pudl.convert.epacems_to_parquet:main"
 ferc_to_sqlite = "pudl.ferc_to_sqlite.cli:main"
 datasette_metadata_to_yml = "pudl.convert.datasette_metadata_to_yml:main"
 pudl_datastore = "pudl.workspace.datastore:main"
-pudl_etl = "pudl.cli:main"
+pudl_etl = "pudl.cli.etl:main"
 pudl_setup = "pudl.workspace.setup_cli:main"
 state_demand = "pudl.analysis.state_demand:main"
 pudl_check_fks = "pudl.etl.check_foreign_keys:main"
+pudl_reset_db = "pudl.cli.reset_db:main"
```

Comment:
Out of scope for this PR, but I wonder if we should think about a command tool reorg? We could group types of commands together:

- pudl db check_foreign_keys
- pudl db setup
- pudl etl run
- pudl ferc_to_sqlite run

IDK maybe not worth the time and confusion.

Reply:
Yeah, I think that would be nice - though if we're pulling out `pudl_etl` and `ferc_to_sqlite` and getting rid of the pudl settings, we might find ourselves with a pretty small # of CLI tools soon.